karoo-gp.py

```diff
@@ -590,11 +592,14 @@ def fx_karoo_pause(model):
 # Fit to the data
 gp.fit(X, y)
 
-# TODO: Relocated from BaseGP.__init__(). Need to test/debug
-# if self.kernel == 'p':
-#     self.fx_data_tree_write(self.population.trees, 'a')
-#     sys.exit()
-
+if kernel == 'p':
+    tree = gp.population.trees[0]
+    print(f'\nTree ID {tree.id}')
+    print(f'  yields (raw): {tree.raw_expression}')
+    print(f'  yields (sym): {tree.expression}\n')
+    print(gp.population.trees[0].display(method='viz'))
+    print(gp.population.trees[0].display(method='list'))
+    # self.fx_data_tree_write(self.population.trees, 'a')
```

Comment:
Do we want to still save the files in this case?
If not we can add back the`sys.exit()`.

Reply:
Yes, but `fx_data_tree_write` doesn't exist anymore, and the replacement needs work. It's basically there as a reminder.

There should be a `save/load` PR that implements what we talked about with Kai:
1. A `.json` from which you can re-load an active model
2. A `.csv` of all the trees generated by that model

I was thinking that can be after the `sklearn` PR.